src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -445,8 +446,25 @@ private static void fillMethodModel(MethodModel mModel, String name, ArrayList<S
                 continue;
             }
 
-            // handle multiple lines for description.
-            if (inDescription) {
+            idxStart = comment.toLowerCase().indexOf("@example");
+            if (idxStart != -1 || i == 1) {
+                if (idxStart != -1 && comment.length() >= idxStart + 10) {
```

Comment:
To be honest, without comments, I wasn't totally sure what this conditional was for, but mine seemed to work fine! Could you elaborate on this section?